pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/SortOperator.java

```diff
@@ -82,6 +82,10 @@ public List<MultiStageOperator> getChildOperators() {
     return ImmutableList.of(_upstreamOperator);
   }
 
+  @Override
+  public void cancel(Throwable e) {
+  }
```

Comment:
could you also add tests to cover the overwritten cancel and close methods? 
for example this one seems like if we encountered a SORT at the top it will not populate cancellation downwards?